### PR TITLE
Adds Expanded Prayer Card

### PR DESCRIPTION
**Pull request by redreceipt**
## DESCRIPTION

Screen Shot 2019-05-02 at 9 55 01 AM

### How do I test this PR?

---

> I am affirming this is my _best_ work ([Ecclesiastes 9:10](https://www.bible.com/bible/97/ECC.9.10.MSG))

## TODO

- [ ] PR has a relevant title that will be understandable in a public changelog (ie...non developers)
- [ ] Closes [DEV-XXX](#url)
- [ ] No new warnings in tests, in storybook, and in-app
- [ ] Upload GIF(s) of iOS and Android if applicable
- [ ] Set two relevant reviewers

## REVIEW

**Manual QA**

- [ ] Manual QA on iOS and ensure it looks/behaves as expected
- [ ] Manual QA on Android and ensure it looks/behaves as expected

**Code Review: Questions to consider**

- [ ] Read through the "Files changed" tab _very carefully_
- [ ] Edge cases: what assumptions are made about input?
- [ ] What kind of tests could be written?
- [ ] How might we make this easier for someone else to understand?
- [ ] Could the code be simpler?
- [ ] Will the code be easy to modify in the future?
- [ ] What's one part of these changes that makes you excited to merge it?

---

> The purpose of PR Review is to _improve the quality of the software._

### CODE REVIEW

**Comment by IsaacHardy** [comment]
@redreceipt Is there anyway to give it a bitmore length?

File: packages/newspringchurchapp/src/prayer/PrayerList/index.js [hunk 1]
```diff
@@ -129,11 +129,12 @@ class PrayerList extends PureComponent {
...
                   {(flagPrayer) => (
                     <PrayerCard
-                      avatarSize={'medium'}
                       onPress={this.handleOnPress}
+                      expanded
+                      avatarSize={'medium'}
                       options={[
                         {
-                          title: 'Flag as Inappropriate',
+                          title: 'Flag as inappropriate',
```
Comment (richarddubay) [inline packages/newspringchurchapp/src/prayer/PrayerList/index.js:137]: My only pushback here (and why I changed it in a different PR) is that I feel like menu items should be titles not sentences. But that's beyond the scope of getting things done today.

**Comment by richarddubay** [comment]
> @redreceipt Is there anyway to give it a bitmore length?

@IsaacHardy I can't tell if you're joking. That card is literally the whole screen. :)

**Comment by IsaacHardy** [comment]
Screen Shot 2019-05-02 at 11 35 13 AM
This is the reason ^

**Comment by richarddubay** [comment]
Oh you don't want to see that there is another card below it.

**Review by richarddubay — APPROVED** [review]
I'm good with this.

**Review by IsaacHardy — APPROVED** [review]
LGTM

**Pull request merged**